src/scripts/install_machine.py

```diff
@@ -54,11 +54,20 @@ def get_graphics_capabilies(connection):
 
 
 def prepare_graphics_params(connection):
+    # Use ipv6 as workaround for https://bugs.launchpad.net/ubuntu/+source/qemu/+bug/1492621
+    spice_address = '::1'
+    # Check to make sure loopback has an IPv6 address assigned
+    # if not default back to IPv4
+    ipv6_loopback = subprocess.check_output(['ip', '-6', 'addr'])
+
+    if not "lo:" in str(ipv6_loopback):
+        spice_address = '127.0.0.1'
```

Comment:
Can you please do `ip -6 addr show dev lo`? And make this robust against failures of the call? The `ip` utility may not be installed, or perhaps some other weird thing. We don't want the script to crash then.

Reply:
Yep! Thanks for the review, I've just reworked it quite a bit to be more robust and check against ifconfig if ip doesn't exist. This should be alot more robust